libr/core/cmd_print.c

```diff
@@ -4038,9 +4038,9 @@ static int cmd_print(void *data, const char *input) {
 		case 'j': // "pIj" is the same as pDj
 			if (l != 0) {
 				if (input[2]) {
-					cmd_pDj (core, input + 2);
+					cmd_pDj (core, input + 2, block);
```

Comment:
Imho cmd_pDj should not get block as argument, and if it does it should be block and lentgh, not user input